src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
If we're going to the extent of supporting notifications in SW I think we should probably also detect support for push manager. I'm not sure why calling it earlier should be an issue, as the primary purpose of the method would be to determine if the agent supports the necessary API's for push notifications to work.